python/ppml/src/bigdl/ppml/fl/nn/tensorflow/aggregator.py

```diff
@@ -119,3 +120,9 @@ def sort_by_key(kv_tuple):
             invalidInputError(False,
                               f'Invalid phase: {phase}, should be train/eval/pred')
 
+    def load_uploaded_model(self, client_id, model_path):
+        if self.model is not None:
+            logging.warn(f"Model exists, model uploading from {client_id} ignored.")
```

Comment:
Only one client can specify server model and server model path; you need to check that, and report error (instead of warning)